File: app/src/main/java/com/neome/api/meta/base/dto/StudioVarValueSetOfUser.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdForm;
import com.neome.api.meta.base.Types.MetaIdPlugin;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioVarValueSetOfUser extends StudioBase
{
  @Nullable
  public StudioMapOfUserCondition node;

  @Nullable
  public MetaIdForm sourceFormId;

  @Nullable
  public MetaIdPlugin sourcePluginId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioVarValueSetOfUser.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdPlugin
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioMapOfUserCondition

interface StudioVarValueSetOfUser : StudioBase
{
  val node: StudioMapOfUserCondition?
  val sourceFormId: MetaIdForm?
  val sourcePluginId: MetaIdPlugin?
}